src/device/index.js

```diff
@@ -640,14 +640,44 @@ realityEditor.device.disablePinchToScale = function() {
     this.editingState.pinchToScaleDisabled = true;
 };
 
+/**
+ * This system allows any number of modules to mark with a flag that they're
+ * currently claiming the pointer events for a camera control mode.
+ * @type {Object.<string, boolean>}
+ */
+realityEditor.device.manualCameraControlFlags = {};
```

Comment:
nit: given that every entry in this will be set to true and never false, I think it makes more sense to be a list of names rather than an object. Feel free to ignore though!

Reply:
My thought was that making it an object would inherently prevent duplicates, but I do see what you mean

Reply:
We could also get fancy and use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set

Reply:
Good idea, updated! I forgot that JS has a Set type, this is a nice place to use it